tests/integration/test_results.py

```diff
@@ -22,6 +23,12 @@ def test_general():
     utime = rusage["ru_utime"]
     utime = utime["tv_sec"] + 1e-6 * utime["tv_usec"]
     assert 0.25 < utime < 0.5
-    assert 1073741824 < rusage["ru_maxrss"] * 1024 < 1100000000
+    if platform.system() == "Darwin":
+        rset_size_bytes = rusage["ru_maxrss"]
+    elif platform.system() == "Linux":
+        rset_size_bytes = rusage["ru_maxrss"] * 1024
```

Comment:
I think it might be better to normalize this field as bytes in the Rust code.  If not, we need to encode the arch or something similar in the res so that someone reading it knows how to interpret this field.  The reader's runtime arch may be different.

Reply:
Given that primary architecture is Linux, I decided to normalize to Kilobytes. Happy to do the reverse, if you feel strongly about it.

Reply:
Yeah, please use bytes.  I think these conventions are hard to keep track of, and irrelevant in the world of arbitrary-size integers.  Let's use the basic units.